src/pudl/output/pudltabl.py

```diff
@@ -241,12 +236,15 @@ def _agg_table_name(self, table_name: str) -> str:
                 table_name = table_name.replace("_AGG", "")
         return table_name
 
-    def _get_table_from_db(self, table_name: str, resource: Resource) -> pd.DataFrame:
+    def _get_table_from_db(
+        self, table_name: str, resource: Resource, update: bool = False
+    ) -> pd.DataFrame:
         """Grab output table from PUDL DB.
 
         Args:
             table_name: Name of table to get.
             resource: Resource metadata used to enforce schema on table.
+            update: Ignored. Retained for backwards compatibility only.
```

Comment:
what does this mean? does `Ignored` mean that it doesn't do anything? but is here bc there may be some things that used `update` in the past?